readme.rst

```diff
@@ -685,13 +685,17 @@ If you would like to clean only one cache type:
 
 .. code-block:: sh
 
-   $ n98-magerun.phar cache:clean [code]
+   $ n98-magerun.phar cache:clean [code] [--reinit --no-reinit]
```

Comment:
I think this should be:

     $ n98-magerun.phar cache:clean [--reinit][--no-reinit][<code>]

This takes care about the two new options and fixes the misnomer on the operand. I'm aware this is not yet consistent style, so perhaps just for the two new options and you might want to separate by space. Please decide on your own.